src/execution/window_segment_tree.cpp

```diff
@@ -204,7 +204,7 @@ unique_ptr<WindowAggregatorState> WindowConstantAggregator::GetLocalState() cons
 }
 
 void WindowConstantAggregator::Evaluate(WindowAggregatorState &lstate, const idx_t *begins, const idx_t *ends,
-                                        Vector &target, idx_t count) const {
+                                        Vector &target, idx_t count, idx_t row_idx) const {
```

Comment:
How does `WindowConstantAggregator` handle exclude? Or is exclude a disqualifying condition?

Reply:
I thought it to be a disqualifying condition and changed `IsConstantAggregate()` accordingly.